perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -83,6 +84,20 @@
                      'Allow swiftclient to access Swift service without \n'
                      'having to verify the SSL certificate')
 
+flags.DEFINE_integer('objects_per_stream', 1000,
+                     'Number of objects to send and/or receive per stream. '
+                     'Only applies to the api_multistream scenario.',
+                     lower_bound=1)
+flags.DEFINE_integer('object_size_kb', 1,
```

Comment:
Just to clarify, this PR does not include object size distribution support yet, right?

(in the future, when we add object size distribution support, how will that play together with the "api_multistream" scenario here?)